### DISCUSSION TITLE
Am I bad at x86 assembly?

### DISCUSSION THREAD

**Discussion by kill3rc** [community/community#147115] [Programming Help]
### Body

[x86 Calculator](https://github.com/DumbestPerson224/CalculatorASM/tree/main/non-type-safe/x86)
The link is a calculator I made a couple weeks ago in x86 assembly language to get better at it. Can you please review it?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hi @DumbestPerson224, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
